.github/workflows/oidc-conformance-test.yml

```diff
@@ -64,7 +64,7 @@ jobs:
          "${asset_url}"
       
     - name: Clone conformance suite
-      run: git clone --depth 1 --branch release-v4.1.4 https://github.com/openid-certification/conformance-suite.git
+      run: git clone --depth 1 --branch release-v4.1.8 https://github.com/openid-certification/conformance-suite.git
```

Comment:
Can we parameterize the release tag value in the script?

It doesn't have to be an input but atleast as a variable